### ISSUE TITLE
XDebug is not working

### ISSUE THREAD

**Issue by moroland** [ironstar-io/tokaido#69]
**What happened**:

Added the following to config.yml:
```
system:
  xdebug:
    port: 9000
    enabled: true
    autostart: true
```

Run `tok up`

Got this:
```
$ tok up

🚀  Tokaido is starting up!

🔄  Creating a background process to sync your local repo into the Tokaido environment

⠦   Tokaido is starting your containersTokaido encountered a fatal error and had to stop at command 'docker-compose -f /Users/__REMOVED__/docker-compose.tok.yml kill fpm'
Named volume "tok_PROJECTNAME_mysql_database:/var/lib/mysql:rw" is used in service "mysql" but no declaration was found in the volumes section.
2018/11/07 17:12:56 exit status 1

```

**What you expected to happen**:

Xdebug to be working :)

**How to reproduce it (as minimally and precisely as possible)**:

Add those to config.yml and run `tok up`.

**Anything else we need to know?**:

**Environment**:
- Tokaido version (use `tok version`): 
Tokaido Version: v1.2.0
Build Date:      2018-09-30T04:18:11Z
Compiler:        go1.10.4
Platform:        darwin/amd64
 
- Drupal version
7
- Operating system and version
OS X

**Additional context**

If I remove the xdebug config and run `tok up` again, it works.

Checking what changed in docker-compose.tok.yml, I noticed that after adding xdebug settings and run tok up, the following part is missing:
```
volumes:
  tok_PROJECTNAME_mysql_database:
    external: true
```

**Comment by otakumike** [comment]
Hey @moroland! We've got a fix in for this, thanks for opening the issue!

I'm not familiar enough with xdebug to say if it's working properly yet, but I am able to get the fpm container to hook back into my vscode debugger. 

Here's my editor config: 

```
{
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Listen for XDebug",
            "type": "php",
            "request": "launch",
            "port": 9000,
            "pathMappings": {
                "/tokaido/site": "/code/umami"
            }   
        },
        {
            "name": "Launch currently open script",
            "type": "php",
            "request": "launch",
            "program": "${file}",
            "cwd": "${fileDirname}",
            "port": 9000,
            "pathMappings": {
                "/tokaido/site": "/code/umami"
            }            
        }
    ]
}
```

And attached is a screenshot of it seemingly working.
![1541888461_0921_11112018_1417x1512](https://user-images.githubusercontent.com/9390978/48307789-cfe5a400-e5a8-11e8-849b-666ecf773f21.png)

**Comment by otakumike** [comment]
(Note that the fix will come out in 1.3.0 later today)

**Comment by otakumike** [comment]
1.3.0 release and xdebug docs updated, going to close this now.
